### DISCUSSION TITLE
Add the pre-defined web components to html tags

### DISCUSSION THREAD

**Discussion by pauleveritt** [koxudaxi/pyscript-pycharm-plugin#11] [Ideas]
Things like `<py-script>`, `<py-env>`, etc. generate an "Unknown html tag" warning. You can alt-enter to add them. Wonder if the plugin could automate that.

These are pre-defined Web Components from the PyScript project. They want to automate the creation of these. In the future, we might be able to recognize the list of "tags" dynamically.

**Comment by koxudaxi** [comment]
@pauleveritt 
I'm sorry for my late reply.
But, I don't forget the project. :)
https://github.com/koxudaxi/pyscript-pycharm-plugin/pull/18

**Reply by pauleveritt** [reply]
Honestly, I wasn't expecting a response any time soon. 😀
